package.json

```diff
@@ -5,7 +5,9 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "cy:open": "cypress open",
+    "cypress": "start-server-and-test start:silent http://localhost:19006/ cy:open"
```

Comment:
can you run `yarn cypress` successfully?

Reply:
now we can